debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/SnapshotReader.java

```diff
@@ -692,6 +692,7 @@ protected void execute() {
                         )
                         .forcedBeat(source.partition(), source.offset(), this::enqueueRecord);
                     // set the filter in the offset
+                    // todo signifier of filters vs config?
```

Comment:
What about this todo? Tacking a step back, how do you plan to address the multiple todo markers?